### ISSUE TITLE
Mutiple BBOX

### ISSUE THREAD

**Issue by eternal-f1ame** [ziqi-jin/finetune-anything#32]
Can I finetune the model using multiple bounding boxes for the same Image 
(eg. in case of KITTI Dataset if I want to finetune the model to segment cars from background and there are multiple instances of cars in the background)
Is there a way I can provide the bounding boxes for all cars in that frame as my prompt or should I iteratively give the bounding box and segmentation of every single instance of a car.

**Comment by ziqi-jin** [comment]
SAM is originally an instance segmentation model. I suggest that all points of the whole image be used as prompts, and the label can use the instance segmentation label of each car.
finetune-anything does not yet support instance segmentation tasks, it will be updated later, thank you for your attention

**Comment by AymanTalkhani** [comment]
@eternal-f1ame We've actually built out a platform that finetunes a research variant of SAM for labeling purposes at [InstaLabel](https://www.instalabel.ai/). All you need to do is upload your dataset, select and label a few samples you'd like to finetune the model on in our platform, and hit the 'Train annotator' button. You can then head over to the 'Annotate' tab, click 'Instant Annotation' and have the finetuned model label the rest of your dataset for you. We also offer crowdsourcing if you'd like to correct the AI predictions for verified ground truth labels. Should be useful if you're interested in you'd like to segment multiple polygons in the same image for an entire dataset!
